src/CMakeLists.txt

```diff
@@ -443,14 +443,14 @@ if (WITH_SERIALIZATION)
     set(fb_header "${fb_dir}/halide_ir_generated.h")
     add_custom_command(
         OUTPUT "${fb_header}"
-        COMMAND flatbuffers::flatc --cpp -o "${fb_dir}" "${fb_def}"
+        COMMAND FlatBuffers::flatc --cpp -o "${fb_dir}" "${fb_def}"
```

Comment:
If I remember correctly, flatbuffer package is pretty strange here that is use `FlatBuffers` as the package name but inside the package the namespace is all lower-case `flatbuffers`

Reply:
To make matters worse, the apt packages on my Google linux box have `FlatBuffers.cmake` but the ones from GitHub Actions are `Flatbuffers.cmake`. Oy. Gonna switch back to the way it was before since that seems to be more likely.

Reply:
yikes that sounds bad